dissect/target/plugins/os/unix/linux/_os.py

```diff
@@ -26,11 +26,10 @@ def detect(cls, target: Target) -> Optional[Filesystem]:
                 fs.exists("/var")
                 and fs.exists("/etc")
                 and fs.exists("/opt")
-                or (fs.exists("/sys") or fs.exists("/proc"))
-                and not fs.exists("/Library")
+                or (fs.exists("/proc") or fs.exists("/sys"))
```

Comment:
Why is this necessary in the first place? Are there scenarios where you _only_ have a `/proc` or `/sys`? Why not make this stricter.

Reply:
Acquire allows for them to be separately collected, so technically a acquire collection of just `/proc` or `/sys` could happen.